frontend/packages/dev-console/src/components/pipelines/PipelineParameters.tsx

```diff
@@ -58,34 +52,14 @@ const PipelineParameters: React.FC<FormikProps<FormikValues>> = ({
         </MultiColumnField>
         <hr />
         {pipelineParameterAccess && (
-          <ButtonBar
-            inProgress={isSubmitting}
+          <FormFooter
+            handleReset={handleReset}
+            isSubmitting={isSubmitting}
             errorMessage={status && status.submitError}
-            successMessage={status && status.success}
-          >
-            {!status && dirty && (
-              <Alert
-                isInline
-                className="co-alert"
-                variant="info"
-                title="The information on this page is no longer current."
-              >
-                Click Reload to update and lose edits, or Save Changes to overwrite.
-              </Alert>
-            )}
-            <ActionGroup className="pf-c-form">
-              <Button
-                type="submit"
-                variant={ButtonVariant.primary}
-                isDisabled={!dirty || !!errors.parameters}
-              >
-                Save
-              </Button>
-              <Button type="button" variant={ButtonVariant.secondary} onClick={handleReset}>
-                Reload
-              </Button>
-            </ActionGroup>
-          </ButtonBar>
+            successMessage={status && !dirty && status.success}
+            disableSubmit={!dirty || !_.isEmpty(errors)}
```

Comment:
disableSubmit should be true when it is dirty or errors is not empty right? then it should be
```
disableSubmit={dirty || !_.isEmpty(errors)}
```

Reply:
@gijohn What I understand is I should keep the Submit button disabled when `there are no changes in the form or there are errors in the form` and should enable it when the user `completes all the required fields`. The value of `dirty` is `true` when changes are made to the form. So the condition that you mentioned would keep the submit button disabled when changes are made to the form.

Reply:
got it. mistaken for error scenario.